src/main/resources/application.yaml

```diff
@@ -60,7 +60,7 @@ idam:
     secret: ${S2S_SECRET_TASK_MONITOR:AAAAAAAAAAAAAAAA}
     name: wa_task_monitor
   s2s-authorised:
-    services: ${WA_S2S_AUTHORIZED_SERVICES:wa_task_monitor}
+    services: ${WA_S2S_AUTHORIZED_SERVICES:wa_task_monitor,wa_task_management_api}
```

Comment:
Does Task Management call Task Monitor?

Reply:
Whitelisted wa-task-management-api as the post-deployment tests makes requests as task management and now needs to manually trigger the jobs.

Reply:
Ideally I think the post deployment ft tests should have their own s2s service is this something we want to explore ?